### ISSUE TITLE
Hi from Jellyfin!

### ISSUE THREAD

**Issue by anthonylavado** [leinelissen/jellyfin-audio-player#1]
Hey there, I'm one of the project leads from Jellyfin. This looks great, and as an iOS user, I'm so happy to see it out there.

I'll give it a shot with some testing soon, but if it looks good, I'll talk to the team about how we can help with any promotion, building/maintenance, or even getting it out to the store.

**Comment by drazil100** [comment]
Ideally I’d prefer it just be one app for all of jellyfin’s features. If this ends up needing to be a separate app (even if it’s just a stepping stone while we wait for the main app to get support) I like the name “Jellyfin Music” as it is in line with Apple Music, YouTube Music, and Amazon Music.

**Comment by anthonylavado** [comment]
@drazil100 We're all for options (I mean, just look at all the apps Plex had these days 😂).

The idea is indeed to have one main app going forward, but that's a long road in our type of volunteer based open source. Until then, we're happy to help anywhere in the community.

**Comment by drazil100** [comment]
Yeah I am totally for a music only app. Could even potentially include features the normal jellyfin app and jellyfin web would not support/need like the ability to not only download an offline library but ability to keep it up to date with changes to the online copy. Also maybe the ability to do gapless playback so the next song starts loading before the previous song ends. There are a lot of good reasons to have a dedicated app.

**Comment by leinelissen** [comment]
I think there are good reasons for having separate apps since the use cases are somewhat different for video vs audio. This is the whole reason I built this app: the web app works perfectly for whenever I need to watch something, but background audio is a major requirement when I'm listening on the go or playing stuff on my home soundsystem.

These could be unified in a single app, but this also puts a lot of weight on this app. It is no suprise that for iOS, Apple decided to separate these functionalities into a Music and TV app, even treating podcasts differently from music.

**Comment by leinelissen** [comment]
@anthonylavado @drazil100 Thank you for the support by the way! I'm happy to build out a product for this niche, and if it helps the ecosystem, I'm glad to help out as well.

**Comment by drazil100** [comment]
I'm just happy to see this niche get some attention. I actually first installed jellyfin to try it out on my music library. I ended up ripping all my movies later but I still have a need for a solution I can use to replace both iTunes and google play music. Running Linux I don't really have the option to install iTunes and google play music is at its best extremely buggy and going to be shut down in favor of youtube music (whenever google gets around to it). I needed SOMETHING to replace these things and now that ALL of my media is on jellyfin getting jellyfin up to spec for music is my best option. As product primarily centered around video I didn't think anyone was ever going to get around to music though. Thank you very much! :)

**Comment by Darkwolf1515** [comment]
I have to agree with a separate music app, usually when I want to play something on the go, I wanna flick on an app, have it launch straight into the music, have it all feel smooth and native. Something I hated about plex was that to get to my downloaded music, I had to open the app, wait for it to fail to connect, and then I could see my downloaded music. 

Some things like books/audiobooks/comics, and music, just need their own native apps dedicated to them.

Hope we can get this onto the app store, and some download support to make it perfect!
